test/integration/targets/mongodb_shard/aliases

```diff
@@ -0,0 +1,5 @@
+destructive
+shippable/posix/group1
+skip/osx
+skip/freebsd
```

Comment:
To efficiently skip RHEL in CI, add:

```
skip/rhel
```

Reply:
Done. I hope to get all RedHat variants working at some point though.